src/components/PremiumTodos.js

```diff
@@ -1,38 +1,65 @@
+
+export function PremiumTodos() {
+    const { user, signOut } = useAuthenticator((context) => [context.user]);
 
-export function PremiumTodos (){
     const [listTodos, setListTodos] = useState([]);
     const [showTodosDone, setShowTodosDone] = useState(null);
-
     useEffect(() => {
-        const subscription = DataStore.observeQuery(Todo).subscribe(msg => {
+        DataStore.observeQuery(Todo).subscribe(msg => {
             setListTodos(msg.items);
-          });
-
-    },[]);
+        });
+    }, []);
 
-    const displayTodosDone = ()=> {
+    const displayTodosDone = () => {
         setShowTodosDone(!showTodosDone)
     }
-   
+
     return (
         <div className='card'>
             <div className='header'>Todoco</div>
-                <TodosInput listTodos={listTodos}/>
-                <Todos listTodos={sortTodos(listTodos.filter((todo)=> !todo.done ))}/>
-                <div className='listTodosDone' onClick={displayTodosDone}>Completed ({filterTodos(listTodos).length})
-                    {filterTodos(listTodos).length !== 0 ? <ShowTodosDone className='iconShowTodosDone' d={showTodosDone ? "M3 4.5h14.25M3 9h9.75M3 13.5h5.25m5.25-.75L17.25 9m0 0L21 12.75M17.25 9v12" :
-                        "M3 4.5h14.25M3 9h9.75M3 13.5h9.75m4.5-4.5v12m0 0l-3.75-3.75M17.25 21L21 17.25"}
-                        onClick={displayTodosDone}/> : ""}
+            <TodosInput listTodos={listTodos} />
+            <Todos listTodos={sortTodos(listTodos.filter((todo) => !todo.done))} />
+            <div className='listTodosDone' onClick={displayTodosDone}>Completed ({filterTodos(listTodos).length})
+                {filterTodos(listTodos).length !== 0 ? <ShowTodosDone className='iconShowTodosDone' d={showTodosDone ? "M3 4.5h14.25M3 9h9.75M3 13.5h5.25m5.25-.75L17.25 9m0 0L21 12.75M17.25 9v12" :
+                    "M3 4.5h14.25M3 9h9.75M3 13.5h9.75m4.5-4.5v12m0 0l-3.75-3.75M17.25 21L21 17.25"}
+                    onClick={displayTodosDone} /> : ""}
+            </div>
+            {showTodosDone && <Todos listTodos={filterTodos(listTodos)} />}
+
+            <div>
+                {user && <div>
+                    <Button
+                        onClick={async () => {
+                            await DataStore.clear();
+                            signOut();
+                        }}
+                        label='Sign out' />
                 </div>
-                {showTodosDone && <Todos listTodos={filterTodos(listTodos)}/>}     
+                }
+
+                {!user &&
+                    <div>
+                        <h2>
+                            Want to save your todocos online?
+                        </h2>
+                        <p>
+                            Saving your tocodos online makes it possible to access them from
+                            anywhere with any device.
+                        </p>
+                        <Link to="/login"><Button label='SignIn' /></Link>
+                    </div>
```

Comment:
Move this to a component called `<LoginAdvantages>`
The end result should be something like this:
```js
{!user && <LoginAdvantages/>}
```
You can give a better if you find one.